lib/index.js

```diff
@@ -77,6 +78,16 @@ AxeBuilder.prototype.withTags = function (tags) {
 
   return this;
 };
+/**
+ * Configure aXe before running analyze. *Does not chain.*
+ * @param  {Object} config Configuration object to use in analysis
+ */
+AxeBuilder.prototype.configure = function(config) {
+  if (typeof config !== 'object') return;
```

Comment:
This will end up throwing an exception if chained. Maybe we should throw an exception that gives more information instead of just returning undefined?